README.md

```diff
@@ -65,79 +67,70 @@ C'est tout le framework du Pass Culture!
 - `http://localhost:3001/` devrait être lancé et fonctionnel
 - Connectez-vous avec `pctest.admin93.0@example.com` et `user@AZERTY123`
 
-### Configurer son API localement
+#### Flask Admin
 
-Les actions suivantes sont requises afin de faire fonctionner l'IDE correctement côté API et de pouvoir lancer les `hooks` de `pre-commit`
-
-- Installer Python 3.9
-- `cd pass-culture-main/api`
-- `python3 -m venv venv`
-- `source venv/bin/activate`
-- `pip install -r requirements.txt`
-Troubleshooting:
-
-if you got errors, during requirements installation, about psycopg, this may help you: https://stackoverflow.com/questions/9678408/cant-install-psycopg2-with-pip-in-virtualenv-on-mac-os-x-10-7/62931654#62931654
-
-- `pip install -e .`
-- Installation de nltk:
-  - Version Ubuntu: `python -m nltk.downloader punkt stopwords`
-  - Version Mac: `pip install nltk`
+- lancer l'`api`
+- lancer `pro` ou `webapp`
+- se connecter avec les identifiants d'un compte admin
+- visiter `http://localhost/pc/back-office/`
 
 ### Exécution des tests (API, WebApp, Pro)
 
-- Point d'attention : l'API et le pro doivent être lancés pour pouvoir lancer les tests
 - API
   1. `pc start-backend`
   2. `pc test-backend` (permet de lancer tous les tests de l'API)
   3. `pc test-backend <path_to_test_file> -k <test_name>` (permet de lancer des tests spécifiques)
   4. `pc test-backend <path_to_test_file> -x <test_name>` (permet de lancer des tests spécifiques et d'arrêter l'exécution au premier test fail)
 - WEBAPP / PRO
   1. `yarn test:unit`
-  2. `yarn test:cafe` (attention, il doit y avoir des données dans la sandbox au préalable `pc sandbox -n testcafe`)
+  2. `yarn test:cafe` (tests end2end)
+  - le backend (`api`) doit être lancé
+  - il doit y avoir des données dans la sandbox au préalable `pc sandbox -n testcafe`
 
-## Développeurs.ses
+## Développement
 
 ### Environnement python local
 
-Pour pouvoir lancer les `hooks` de `pre-commit` sur le projet API, il faut installer l'environnement python en local.
+- Installer Python 3.9 et `pip`
+- Une possibilité est de monter un [virtualenv](https://python-guide-pt-br.readthedocs.io/fr/latest/dev/virtualenvs.html) afin d'avoir un environnement isolé et contextualisé pour les besoins de l'API
 
-- installer Python 3.9 et `pip`
-- monter un [virtualenv](https://python-guide-pt-br.readthedocs.io/fr/latest/dev/virtualenvs.html) afin d'avoir un environnement isolé et contextualisé pour les besoins de l'API
   1. `pip install virtualenv`
   2. `cd pass-culture-main/api`
   3. `python -m venv venv`
   4. `source venv/bin/activate`
   5. `pip install -r requirements.txt`
 
-### Rebuild
+- Il est également possible d'utiliser [pyenv](https://github.com/pyenv/pyenv) pour sélectionner un interpreteur python de votre OS. Avantage : ne pas à avoir à lancer le virtualenv
```

Comment:
Pourquoi ne pas utiliser de virtualenv si on utilise pyenv ?

Reply:
car `pyenv` te fournit un env python local

Reply:
Il me semble que [ce n'est pas le cas](https://github.com/pyenv/pyenv#in-contrast-with-pythonbrew-and-pythonz-pyenv-does-not).